asset/cpu_triggered_asset_utilization.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import sys
+import time
+import winreg
+
+LOGGED_USER_NAME = getpass.getuser()
+
+parser = argparse.ArgumentParser(description = 'CPU usage triggered asset utilization.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
I would say, put all global code into a function, eg `main()`.

Then call that function like this:
```
if __name__ == '__main__':
    main()
```